File: Cosmos-Android/app/src/main/java/wannabit/io/cosmostaion/dao/OkTicker.java
package wannabit.io.cosmostaion.dao;

import com.google.gson.annotations.SerializedName;

public class OkTicker {
    @SerializedName("close")
    public String close;

    @SerializedName("high")
    public String high;

    @SerializedName("low")
    public String low;

    @SerializedName("open")
    public String open;

    @SerializedName("price")
    public String price;

    @SerializedName("product")
    public String product;

    @SerializedName("symbol")
    public String symbol;

    @SerializedName("timestamp")
    public String timestamp;

    @SerializedName("volume")
    public String volume;
}

File: Cosmos-IOS/Cosmostation/Cosmostation/Dao/OkTicker.swift
//
//  OkTicker.swift
//  Cosmostation
//
//  Created by 정용주 on 2021/02/07.
//  Copyright © 2021 wannabit. All rights reserved.
//

import Foundation

public struct OkTicker {
    var close: String?
    var high: String?
    var low: String?
    var open: String?
    var price: String?
    var product: String?
    var symbol: String?
    var timestamp: String?
    var volume: String?
    
    init(_ dictionary: NSDictionary?) {
        self.close = dictionary?["close"] as? String
        self.high = dictionary?["high"] as? String
        self.low = dictionary?["low"] as? String
        self.open = dictionary?["open"] as? String
        self.price = dictionary?["price"] as? String
        self.product = dictionary?["product"] as? String
        self.symbol = dictionary?["symbol"] as? String
        self.timestamp = dictionary?["timestamp"] as? String
        self.volume = dictionary?["volume"] as? String
    }
}
